app-ios/Native/Sources/Extension/Room+Extension.swift

```diff
@@ -50,27 +48,25 @@ extension Room {
 
     public var floorInfo: String {
         switch type {
-        case .roomF, .roomG:
+        case .roomJ, .roomK:
             return "1F"
-        case .roomH, .roomI, .roomJ, .roomIJ:
+        case .roomL, .roomM, .roomN:
             return "B1F"
         }
     }
 
     public var iconName: String {
         switch type {
-        case .roomF:
+        case .roomJ:
+            return "ic_square"
+        case .roomK:
             return "ic_rhombus"
-        case .roomG:
+        case .roomL:
             return "ic_circle"
-        case .roomH:
+        case .roomM:
             return "ic_diamond"
-        case .roomI:
-            return "ic_square"
-        case .roomJ:
+        case .roomN:
```

Comment:
```suggestion
public var iconName: String {
    switch type {
        case .roomJ:
            return "ic_square"
        case .roomK:
            return "ic_diamond"
        case .roomL:
            return "ic_circle"
        case .roomM:
            return "ic_rhombus"
        case .roomN:
```

I think this is the correct mapping.

Reply:
I missed 🙏🏼 
I fixed!